misc/module_config.json

```diff
@@ -1,10 +1,12 @@
 {
   "modules": {
     "pass": {
-      "enabled": true
+      "enabled": true, 
+      "timeout": 60
```

Comment:
yah, I think these should be timeout_sec to avoid confusion. Also the default is default_timeout_sec so it makes it consistent.